### PR TITLE
publish "latest" image with passed proposals

### PR DESCRIPTION
**Pull request by turadg**
closes: #100 

Has some other hygiene commits too

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
I could approve this on the basis of not seeing anything wrong with it, but I'd prefer to defer to someone with more expertise.

File: .github/workflows/ci.yml
Comment (mhofman) [inline .github/workflows/ci.yml:143]: Could we not restore the ability to tag `latest` with the name of the branch too? This was useful for manually triggered jobs. Arguably the fact these manually triggered jobs clobber the `use-*` tags but not `latest` could be confusing.

**Review by mhofman — APPROVED** [review]
I don't fully grasp the docker file / bake changes, but I'll trust they're correct with a manually triggered CI workflow on this PR?

File: .github/workflows/ci.yml
Comment (turadg) [reply .github/workflows/ci.yml:143]: We could. How was it useful?

Regardless I'll treat it as a separate issue/PR to get this in sooner and unblock https://github.com/Agoric/agoric-sdk/pull/8873

**Comment by turadg** [comment]
> I'll trust they're correct with a manually triggered CI workflow on this PR

Confirming with https://github.com/Agoric/agoric-3-proposals/actions/runs/7835252713/job/21380263334

File: .github/workflows/ci.yml
Comment (mhofman) [reply .github/workflows/ci.yml:143]: It was useful to generate special "forked" images, but I only used it for generating use layer images.

**Comment by mhofman** [comment]
> Confirming with https://github.com/Agoric/agoric-3-proposals/actions/runs/7835252713/job/21380263334

Looks like it failed.

Btw, please open a PR on `agoric-sdk` to switch `a3p-integration` to `latest` instead of `main`

**Review by mhofman — CHANGES_REQUESTED** [review]
I just remembered that we need to update the default `fromTag` in `defaultConfig` in `build.ts`

**Comment by turadg** [comment]
> PR on `agoric-sdk` to switch `a3p-integration` to `latest` instead of `main`

Certainly. I linked to it [earlier](https://github.com/Agoric/agoric-3-proposals/pull/99#discussion_r1483533227)

> we need to update the default fromTag in defaultConfig in build.ts

Good catch. Amended.

I force pushed because we don't have autofix in this repo.

**Comment by mhofman** [comment]
> > PR on `agoric-sdk` to switch `a3p-integration` to `latest` instead of `main`
> 
> Certainly. I linked to it [earlier](https://github.com/Agoric/agoric-3-proposals/pull/99#discussion_r1483533227)

No I mean that PR also need to switch the `fromTag` in `a3p-integration`'s `package.json`

**Review by mhofman — APPROVED** [review]

**Comment by mhofman** [comment]
Approved on the condition that https://github.com/Agoric/agoric-3-proposals/actions/runs/7838630464/job/21390363902 passes

**Pull request merged**
